README.md

```
@@ -24,35 +24,31 @@ Currently we only support **Pocket Option** (quick trading) with the following f
 * Add support for other trading platforms like Expert Options
 
 # Python 
 For the full python documentation check [here](https://chipadevteam.github.io/BinaryOptionsTools-v2/python.html).
-To install it use pip:
+To install it using pip:
 ```
-pip install binaryoptionstoolsv2==0.1.8
+pip install binaryoptionstoolsv2
+```
+To install it manually:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)